### ISSUE TITLE
lab 0.24.0, 0.25.0 failed due to `ls-files` not exist

### ISSUE THREAD

**Issue by chenrui333** [zaquestion/lab#833]
not sure what was changed, but the most recent upgrade efforts ([0.24.0](https://github.com/Homebrew/homebrew-core/pull/96833) and [0.25.0](https://github.com/Homebrew/homebrew-core/pull/102493)) for homebrew formula failed due to command missing. Thanks!

error log as below:

```
  ==> git add haunted house
  ==> git commit -a -m Initial Commit
  [master (root-commit) e5075ad] Initial Commit
   2 files changed, 0 insertions(+), 0 deletions(-)
   create mode 100644 haunted
   create mode 100644 house
  ==> /opt/homebrew/Cellar/lab/0.24.0/bin/lab ls-files
  2022/03/14 07:33:38 root.go:201: No default remote found
  Error: unknown command "ls-files" for "lab"
  Run 'lab --help' for usage.
```

**Comment by bmeneg** [comment]
Hi @chenrui333 , `ls-files` is a git command and not a lab one. Until PR #809 (released with 0.25.0) the passthrough feature of lab was removed, meaning that commands not recognized by lab are no longer passed to git.

Another note is that the 0.25.0 tag is somewhat bogus release-wise, I'll probably have to delete it and recreate it due to a testing issue caused by misbehavior of our live testing repo :(. Stay tuned.

**Comment by bmeneg** [comment]
@chenrui333 feel free to review PR #836.
